### ISSUE TITLE
aria-allowed-role should ignore hidden roles

### ISSUE THREAD

**Issue by WilcoFiers** [dequelabs/axe-core#1194]
I just found this case in the wild:

```html
<div style="display:none">
  <button role="presentation">Some button</button>
</div>
```

Adding that role is completely pointless, but it isn't an accessibility issue. We should mark these things for review IMO.

**Comment by marcysutton** [comment]
It would fall under a "ghost controls" rule as well, due to the aria-hidden and tabindex on it. Definitely an accessibility issue.

**Comment by jkodu** [comment]
@WilcoFiers, @marcysutton 

Well, in today's implementation, we flag this as a violation with the message - `role presentation  is not allowed for given element`.

Should we instead mark it as `needs review`?

**Comment by jkodu** [comment]
PR Merged. Closing issue.
